edenai_apis/apis/clarifai/clarifai_api.py

```
@@ -342,6 +344,69 @@ def image__object_detection(
                 standardized_response=ObjectDetectionDataClass(items=items),
             )
 
+    async def image__aobject_detection(
+        self, file: str, file_url: str = "", model: Optional[str] = None, **kwargs
```

Ruff (0.14.4)

Unused method argument: `file_url`

(ARG002)